terraform/applications/pastebin/variables.tf

```diff
@@ -0,0 +1,29 @@
+variable "cost_center" {
+  default = "1410"
+  type    = string
+}
+
+variable "environment" {
+  default = "prod"
+  type    = string
+}
+
+variable "project" {
+  default = "pastebin"
+  type    = string
+}
+
+variable "project_desc" {
+  default = "paste.allizom.org"
```

Comment:
Should the prod description be the prod domain `paste.mozilla.org`, or is that queued up for later once we've migrated? (Or something mozit.cloud?)

Reply:
you're right, this is copypasta.

Reply:
this should be repaired.